graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/KotlinCodeGenTest.kt

```diff
@@ -1670,6 +1670,481 @@ class KotlinCodeGenTest {
         assertCompilesKotlin(dataTypes)
     }
 
+    @Test
```

Comment:
Since most of these tests are verifying the annotation on either the generated class or the field, is it possible to avoid string comparing the entire class ? We have a lot of tests above that specifically verify fields, so you can check if the class and specific fields have the required annotations etc. It's much easier to maintain vs trying to get the string just right.

Reply:
Yes that makes sense. I will update the tests as per your suggestions.

Reply:
Thanks.

Reply:
@srinivasankavitha I have updated the unit tests as per your suggestions. Could you take a look at them?

Reply:
Thanks for the changes, they look good. Will merge the PR.